src/types/WindowUiaProviderBase.cpp

```diff
@@ -8,11 +8,12 @@
 
 using namespace Microsoft::Console::Types;
 
-WindowUiaProviderBase::WindowUiaProviderBase(IUiaWindow* baseWindow) :
-    _signalEventFiring{},
-    _baseWindow{ baseWindow },
-    _cRefs(1)
+#pragma warning(suppress : 26434) // WRL RuntimeClassInitialize base is a no-op and we need this for MakeAndInitialize
+HRESULT WindowUiaProviderBase::RuntimeClassInitialize(IUiaWindow* baseWindow) noexcept
 {
+    _signalEventFiring.clear();

```

Comment:
probably don't need to clear this. RuntimeClassInitialize isn't called multiple times, and the object was just constructed with an empty map.